extras/ffmpeg-decoder/build.rs

```diff
@@ -38,6 +38,7 @@ fn main() {
 
     let bindings = bindgen::Builder::default()
         .header("src/bindings.h")
+        .clang_args(["-I/usr/include/ffmpeg/"])
```

Comment:
I'm amazed I managed to build without this on my systems...

This assumes that the ffmpeg include files will be at this fixed location, which is not a given. I think a sounder way to achieve this would be to stop the results of the `Config::new()..probe()` calls above, and call `include_paths()` on them. Aggregate them and remove duplicates, and you should have a list of paths to pass to `-I`.